src/components/AnnotationBrowser/AnnotationCSVDialog.vue

```diff
@@ -230,7 +238,17 @@ export default class AnnotationCsvDialog extends Vue {
         switch (typeof value) {
           case "object":
           case "undefined":
-            row.push("-");
+            switch (this.undefinedHandling) {
+              case "na":
+                row.push("NA");
+                break;
+              case "nan":
+                row.push("NaN");
+                break;
+              default: // 'empty'
+                row.push("");
+                break;
+            }
```

Comment:
This could be cleaner by using a Map or an object

Reply:
Done, thanks!